examples/platform/silabs/SiWx917/SiWx917/wfx_rsi_host.cpp

```diff
@@ -223,7 +198,7 @@ void wfx_setup_ip6_link_local(sl_wfx_interface_t whichif)
      * TODO: Implement IPV6 setup, currently in wfx_rsi_task()
      * This is hooked with MATTER code.
      */
-    SILABS_LOG("%s: warning: not implemented.", __func__);
+    return;
```

Comment:
this function is a void return, you are at the end of the function and it is also empty. it doesn't need a return.